client/app/layouts/splash.html

```diff
@@ -21,7 +21,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <heading align="center">Foodsaving<br>Worldwide</heading>
```

Comment:
Agreed - we should reserve custom names for angular. Maybe something for the `contribute.md` (which I didn't have a look inside for quite some time)